<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="camera_name" value="head_camera" />
    <param name="topic_name" value="image_raw0" />
    <param name="camera_product_id" value="f580" />
    <param name="camera_vendor_id" value="2b03" />
    <param name="usb_port_id" value="usb-0000:00:14.0-1" />
    <param name="dump_directory" value="/home/ad/ad/dump/" />
    <param name="image_width" value="3840" />
    <param name="image_height" value="1080" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
</launch>
